1001-2000/LIT1928Mashaf.xml

```diff
@@ -25,8 +25,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </availability>
          </publicationStmt>
          <sourceDesc>
-            <p>Work of the literatures of Ethiopia and Eritrea</p>
```

Comment:
Both statements should stay, the belong to the basic documentation.

Reply:
Thanks, I will remember.